examples/diffusion/python_stable_diffusion_21/requirements.txt

```diff
@@ -24,4 +24,5 @@
 accelerate
 diffusers
 optimum[onnxruntime]
```

Comment:
I was using ubuntu 22.04 Python 3.10.  pip -i install requirements.txt fails with it complaining about optimum and ORT incompatibilities

Reply:
looks like it's fine when using the existing migraphx docker container along with virtual environment